packages/binding-coap/src/coap-client.ts

```diff
@@ -44,9 +44,6 @@ export default class CoapClient implements ProtocolClient {
         // WoT-specific content formats
         coap.registerFormat(ContentSerdes.JSON_LD, 2100);
```

Comment:
Another point: I guess there are no plans to actually register the JSON-LD content-format for IANA, right?

Reply:
Not as far I know 🙈